boards/qorvo/decawave_dwm3001cdk/decawave_dwm3001cdk_defconfig

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+CONFIG_PINCTRL=y
```

Comment:
```suggestion
```

Reply:
This was actually causing me problems in some contexts, normally it's enabled by the spi stuff but if I'm not using spi it doesn't get enabled, and then the build fails

Reply:
If someone does not enable it then it's a bug, can you post the build error?